performance-timeline/po-disconnect.html

```diff
@@ -30,15 +30,16 @@
 
   async_test(function (t) {
     var observer = new PerformanceObserver(
-        t.step_func(function(entryList, obs) {
-          checkEntries(entryList.getEntries(),
-            [{ entryType: "mark", name: "mark1"}]);
-          t.done();
+        t.step_func(function (entryList, obs) {
+          assert_unreached("This callback must not be invoked");
```

Comment:
Correct, Chrome would fail this particular test. However, this test matches both the spec and Firefox's implementation today. I'm fine with waiting until the spec is updated; it it goes the direction of adding takeRecords, I suspect this test will still be valid.

Chromium's auto-sync process not importing failing tests sounds like an issue / policy to be solved by the chromium project. Inevitably there will be correct web-platform-tests that a browser will fail to pass. I'm not sure that concern is relevant to the review of adding new tests to web-platform-tests if the tests themselves are correct. That said, I am new to this process.